docs/styled.md

```diff
@@ -47,6 +48,26 @@ render(
 )
 ```
 
+### Default Props
+
+Default props can be set via the second argument of `styled`. 
+
+`this.props` is merged with and overwrites any overlapping keys in `defaultProps`.  
+
+*Does not work with shorthand syntax*
+
+```jsx live
+const MyComponent = styled('input', {
+  props: {
```

Comment:
seems like `defaultProps` instead of just `props` would nicely relate to true React APIs

Reply:
I agree with `props` not being optimal. I like @mitchellhamilton's idea of using `withProps` to match recompose.